### PR TITLE
Make the MACE fit more flexible and build a finetuning workflow

### LINKED ISSUE #178: Provide finetuning option for phonons based on MACE-MP-0
Add "--foundation_model=$path_model" to the fitting procotol of mace to also enable automatic fine-tuning with the phonon workflow.

I will play around with this in the next weeks. I think this should just be few changes in the current code.

### PR DESCRIPTION
**Pull request by JaGeo**
Closes #178 

- [x] make all keywords available for the fit (half done - check closely if the current implementation makes sense)
- [x] check function parameters and see if we need more/less or need to think about an overwriting strategy
- [x] build a fine-tuning workflow
- [x] Set useful default settings
- [x] Seperate forcefields and make sure that they can be used separately (will be important for fine-tuning applications that need the current MACE main branch...)

### CODE REVIEW

File: docs/user/fitting/fitting.md [hunk 1]
```diff
@@ -212,6 +212,57 @@ complete_flow = CompleteDFTvsMLBenchmarkWorkflow(
     ...
 )
 ```
+### Finetuning MACE-MP-0
+
+It is also possible to finetune MACE-MP-0. To do so, you need to install MACE-torch 0.3.7. 
+Currently, this can only be done by cloning the git-repo and installing it from there: []()
+
+It is now important that you switch off the default settings for the fitting procedure (use_defaults_fitting=False).
+If one wants to perform very low-data fine-tuning (less than 4 additional structures), please also think about switching off `preprocessing_data`
```
Comment (JaGeo) [inline docs/user/fitting/fitting.md:221]: @QuantumChemist can you confirm that this is enough to skip the train, test split and simply take all data?

Comment (QuantumChemist) [reply docs/user/fitting/fitting.md:221]: But then the MLIPMaker will only run if train.extxyz and test.extxyz files are present in the database_dir
https://github.com/autoatml/autoplex/blob/fdbee8ec41360773ac26fead37e437dc39586180/autoplex/fitting/common/flows.py#L182

Comment (JaGeo) [reply docs/user/fitting/fitting.md]: Yeah. I will only add the option not change the default. This is what I have done for the rest of the implementations here as well. The defaults have not changed.

**Comment by JaGeo** [comment]
@YuanbinLiu FYI, I will merge this and then push a few tiny changes to #203 to fix the merge conflicts. I have prepared everything in #213 already and this should not cause any problems for your pull request.

**Pull request merged**